Makefile

```diff
@@ -55,11 +55,12 @@ BUILDIMAGE_PKGSRC = nginx-1.14.2 # used by tests to approximate mako
 AGENTS = amon config registrar
 
 .PHONY: all
-all: $(STAMP_NODE_PREBUILT) $(STAMP_NODE_MODULES) install
+all: $(STAMP_NODE_PREBUILT) release
```

Comment:
Personally I'd drop `release` from being a dep for `all`. Other components typically don't do that. The Jenkinsfile typical build call is:
```
make print-BRANCH print-STAMP all release publish buildimage bits-upload
```

Reply:
sold.